repo_tests/ambient_git_vars_mirror_test.py

```
@@ -0,0 +1,85 @@
+    """The claim the comment made, now able to fail."""
+    shell, python = _shell_vars(), _python_vars()
+    only_python = sorted(python - shell)
+    only_shell = sorted(shell - python)
+    assert not (only_python or only_shell), (
+        "the ambient-git-var scrub lists have diverged, so a git subprocess is "
+        "unscrubbed on one side (#15176, #15877):\n"
+        f"  only in autobot_shared/paths.py : {only_python or '—'}\n"
+        f"  only in scripts/lib/git-root.sh : {only_shell or '—'}\n"
+        "Add the missing name to BOTH; whichever side is short runs git with the "
+        "ambient variable still set."
+    )
```

Ruff (0.16.3)

[warning] Assertion should be broken down into multiple parts

(PT018)